ETHCHAT.md

```diff
@@ -0,0 +1,201 @@
+---
```

Comment:
this file should go into content/docs/rfcs/70 as README.md, and create a corresponding index in content/menu/index.md